src/api/CreateChecklistItems.ts

```diff
@@ -0,0 +1,113 @@
+import { spWebContext } from "providers/SPWebContext";
+import { IInRequest } from "api/RequestApi";
+import { EMPTYPES } from "constants/EmpTypes";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+const createInboundChecklistItems = (request: IInRequest) => {
+  const [batchedSP, execute] = spWebContext.batched();
+
+  const checklistItems = batchedSP.web.lists.getByTitle("CheckListItems");
+
+  // Welcome Package
+  // Required for all inbounds?
+  checklistItems.items.add({
+    Title: "Welcome Package",
+    Description: "",
+    Lead: "Supervisor",
+    RequestId: request.Id,
+  });
+
+  // IA Training
+  // Required for all inbounds?
+  checklistItems.items.add({
+    Title: "IA Training",
+    Description: "",
+    Lead: "Employee",
+    RequestId: request.Id,
+  });
+
+  // Transfer CAC if civilian employee already has one
+  if (
+    request.hasExistingCAC === "yes" &&
+    request.empType === EMPTYPES.Civilian
+  ) {
```

Comment:
This particular case should never happen, as we only ask CTR if they have existing CAC.